common/flatpak-dir.c

```diff
@@ -2800,9 +3088,72 @@ flatpak_dir_pull_untrusted_local (FlatpakDir          *self,
   if (!ostree_repo_open (src_repo, cancellable, error))
     return FALSE;
 
+  if (gpg_verify)
+    {
+      gpg_result = ostree_repo_verify_commit_ext (src_repo, checksum, NULL, NULL, cancellable, error);
+      if (gpg_result == NULL)
+        return FALSE;
+
+      if (ostree_gpg_verify_result_count_valid (gpg_result) == 0)
+        return flatpak_fail (error, "GPG signatures found, but none are in trusted keyring");
+    }
```

Comment:
(To clarify things: `ostree_repo_verify_commit_ext()` *internally* passes `NULL` for the remote name. We should be using `ostree_repo_verify_commit_for_remote()` instead, I think.